### DISCUSSION TITLE
Not all members of a group acting on commands

### DISCUSSION THREAD

**Discussion by majorsl** [Koenkk/zigbee2mqtt#22268] [Devices]
I have 4 blinds in a Home Assistant group.  All the blinds work fine individually.  I have them in a group because it is a set of 4 windows.  When I trigger actions for the group, it seems like zigbee2mqtt sends the 4 individual commands (too fast?) and the coordinator doesn't get the change to all 4 blinds.  Sometimes 3 out of 4 get it.  Sometimes 2 out of 4.

I'm using a Sonoff Zigbee Dongle - E.  Is there a way to introduce a delay in between commands on z2m so I can see if that might be the cause?  I see there is an "adapter delay" setting, but it was inferred in a post that it is only for deconz adapters.

**Comment by DavidConnack** [comment]
Try to use a zigbee2mqtt goup.
That will send a single command that all the devices will react to.
Less traffic more reliable.

**Reply by majorsl** [reply]
I'd love to because I do that with some lights, but you can't add blinds (covers) to a group.  I raised an issue about it, but then discovered a closed issue that not being able to add them was expected.
